src/content/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Kubernetes Summary page
+tags:
+  - APM
+  - APM UI pages
+  - Kubernetes
+metaDescription: The Kubernetes Summary page provides charts and tables that you can drill down into details about your selected app's performance.
+---
+
+The Kubernetes APM **Summary** page provides general information about the selected cluster, including the age of the current deployment, the number of pods available, CPU and memory usage, pod logs, and metric comparison.
+
+![This is an image of the Kubernetes APM page](./images/kubernetes-apm-ui.png "Kubernetes page")
+
+<figcaption>[one.newrelic.com](one.newrelic.com) > **Explorer** > Select your cluster > **Kubernetes**</figcaption>
+
+<Callout variant="tip">
+  To get a high-level overview of all your applications and services, use the [New Relic explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer).
+</Callout>
+
+## Monitor your Kubernetes cluster [#monitor-kubernetes]
```

Comment:
"Monitor your Deployment within your Kubernetes Cluster" ... that might be too wordy though. But the APM Page doesn't monitor the whole cluster as much as it targets a specific Deployment in the Kubernetes Cluster that is connected to your application.

Reply:
I think it's great to be specific, so I'll change it.